examples/html-css/README.md

```diff
@@ -0,0 +1,17 @@
+## Example: Pure HTMS/CSS
+
+This shows the usage of CMS design system components on a pure HTML/CSS site. Simply include the CSS file in the HTML document.
```

Comment:
Maybe we can include the description of the 3 old examples here, form, article, responsive